### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Nyholm**
This will address some of the changes in #64

### CODE REVIEW

File: HttplugFactory.php [hunk 1]
```diff
@@ -37,7 +37,6 @@ public function find($type)
     {
         $class = $this->findOneByType($type);
 
-        // TODO: use doctrine instantiator?
```
Comment (Nyholm) [inline HttplugFactory.php]: I do not believe it is a good idea to instanciate a factory class without calling its constructor. Our `DiactorosMessageFactory` will not work unless its constructor is used.

Comment (sagikazarmark) [inline HttplugFactory.php]: Agree, remove that comment

**Comment by sagikazarmark** [comment]
Added a few more things to #64

**Comment by Nyholm** [comment]
Added changelog

**Comment by Nyholm** [comment]
:+1:

**Comment by sagikazarmark** [comment]
I added a few things as well. Can you please squash commits. Call it something like Prepare stable release. Thank you.

**Comment by Nyholm** [comment]
Squashed!

**Comment by sagikazarmark** [comment]
Thanks

**Pull request merged**